src/components/atoms/ToggleRP/ToggleRP.scss

```diff
@@ -0,0 +1,32 @@
+@import "colors";
+
+.ToggleRP {
+  height: 2rem;
+  width: 3.2rem;
+  border-color: $ghost-white;
+  border-radius: 45%;
+  background-color: $ghost-white;
+  color: $transparent;
+
+  &:focus {
+    outline: none;
+  }
+}
+
+[role="switch"][aria-checked="true"] :first-child,
```

Comment:
are you sure is cross browsing?

Reply:
I believe it is. This is what I checked with: https://caniuse.com/#search=%5B